src/styles/splitter.less

```diff
@@ -40,10 +40,16 @@
 /*
  * Angular Splitter
  */
-split {
-    split-gutter {
+as-split {
+    .as-split-gutter {
         &:focus {
             .native-focus-outline();
         }
     }
+}
+
+as-split {
+    .as-split-gutter {
+        background-color: #f2f2f2 !important;
```

Comment:
Just noticed that you have a similar rule above, so this background-color property can be moved there I think.

Reply:
Updates, sorry didnt even notice this.